kuflow-temporal-activity-robotframework/pom.xml

```
@@ -39,7 +39,7 @@
   <artifactId>kuflow-temporal-activity-robotframework</artifactId>
   <packaging>jar</packaging>
 
-  <name>KuFlow SDK :: Temporal :: Activity :: Robot Framework</name>
+  <name>KuFlow SDK :: Temporal :: Activity :: RobotFramework</name>
```

```suggestion
  <name>KuFlow SDK :: Temporal :: Activity :: Robot Framework</name>
```
See https://robotframework.org/